python/cuml/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -593,18 +591,11 @@ class RandomForestClassifier(BaseRandomForestModel,
                            return_values=[('dense', '(n_samples, 1)')])
     @cuml.internals.api_base_return_array(get_output_dtype=True)
     @enable_device_interop
-    @deprecate_non_keyword_only(
-        "threshold",
-        "convert_dtype",
-        "predict_model",
-        "layout",
-        "default_chunk_size",
-        "align_bytes",
-    )
     @_handle_deprecated_rf_args('algo', 'fil_sparse_format')
     def predict(
         self,
         X,
+        *,
         threshold = 0.5,
         convert_dtype = True,
```

Comment:
Just as a note -- now that you've converted these to keyword-only parameters, it is possible to standardize the order of those keyword-only parameters across cuML without constituting an API break. That is really nice. For example, you could standardize whether `convert_dtype` should go first or last among the "common" cuML-only parameters. Maybe nice to think about for the future.